application/org.openjdk.jmc.flightrecorder.ui/src/main/java/org/openjdk/jmc/flightrecorder/ui/views/stacktrace/StacktraceView.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018, 2020 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2018, 2020, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Copyright notices are ranges, but comma delimited, in Oracle copyright headers. "Copyright (c) 2018, 2021, ..." would be the appropriate header update.